tests/test_req_vars.py

```diff
@@ -107,6 +108,18 @@ def test_reconstruct_url(self):
 
         self.assertEqual(expected_uri, req.uri)
 
+    def test_nonlatin_path(self):
+        if not six.PY3:
```

Comment:
Consider using the `testtools.skipUnless` decorator to make this a little cleaner, i.e.:

```python
@testtools.skipUnless(six.PY3, 'Test only applies to Python 3')
def test_nonlatin_path(self):
```